server/decorators/expandedDeliusServiceUserDecorator.ts

```diff
@@ -0,0 +1,42 @@
+import { Address, ExpandedDeliusServiceUser } from '../models/delius/deliusServiceUser'
+
+export default class ExpandedDeliusServiceUserDecorator {
+  constructor(private readonly deliusServiceUser: ExpandedDeliusServiceUser) {}
+
+  get address(): string[] {
+    const { addresses } = this.deliusServiceUser.contactDetails
+
+    if (!addresses) {
+      return ['Not found']
+    }
+
+    if (addresses.length === 1) {
+      return this.deliusAddressToArray(addresses[0])
+    }
+
+    // If we have no "from" date, how do we know which is the most recent?
+    if (addresses.every(address => address.from === null)) {
```

Comment:
agree this is a bit iffy. can we log a warning then we know if anyone hits this in prod?

Reply:
Good idea. Do we want it at `warn` or `error` level?

Reply:
erm. yeh error for now.

Reply:
Done